src/charts/SalesChannels.js

```diff
@@ -15,7 +14,7 @@ export default function SalesChannel(selectedYear){
 
   useEffect(()=>{
     const accessToken=getAccessToken();
-    api.get(`http://127.0.0.1:8000/projectcreation-count/${selectedYear.selectedYear}/`, {
+    api.get(`http://127.0.0.1:8000/api/projectcreation-count/${selectedYear.selectedYear}/`, {
```

Comment:
In Api file Axios instance should be something like this:
const api = axios.create({
  baseURL: process.env.REACT_APP_BASE_URL
}); 
and then you do not need to write http:// for each request.
Also in Api.js 
 const response  in getNewRefreshToken function should use api.post not axios.post.